samples/http/sender-protobuf/main.go

```diff
@@ -0,0 +1,47 @@
+//
+//  Copyright 2021 The CloudEvents Authors
+//  SPDX-License-Identifier: Apache-2.0
+//
+
+package main
+
+import (
+	"context"
+	"log"
+
+	pbcloudevents "github.com/cloudevents/sdk-go/binding/format/protobuf/v2"
+	cloudevents "github.com/cloudevents/sdk-go/v2"
+	cehttp "github.com/cloudevents/sdk-go/v2/protocol/http"
+)
+
+func main() {
+	ctx := cloudevents.ContextWithTarget(context.Background(), "http://localhost:8080/")
+
+	p, err := cloudevents.NewHTTP()
+	if err != nil {
+		log.Fatalf("failed to create protocol: %s", err.Error())
+	}
+
+	c, err := cloudevents.NewClient(p, cloudevents.WithTimeNow(), cloudevents.WithUUIDs())
+	if err != nil {
+		log.Fatalf("failed to create client, %v", err)
+	}
+
+	for i := 0; i < 10; i++ {
+		data := &Sample{Value: "sample"}
+		e := cloudevents.NewEvent()
+		e.SetType("com.cloudevents.sample.sent")
+		e.SetSource("https://github.com/cloudevents/sdk-go/v2/samples/http/sender-protobuf")
+		e.SetDataSchema(string(data.ProtoReflect().Descriptor().FullName()))
+		_ = e.SetData(pbcloudevents.ContentTypeProtobuf, data)
```

Comment:
oh! only the data is encoded in protobuf? I had assumed the entire cloudevent was being sent as a protobuf

Reply:
Haven't looked into the code, but maybe line 22 tells the encoder to use protobuf?